lib/tasks/attempts.rake

```diff
@@ -46,6 +46,9 @@ namespace :attempts do
   desc 'Confirm your dev setup is configured properly'
   task check_enabled: :environment do
     failed = false
+    auth_token = IdentityConfig.store.irs_attempt_api_auth_tokens.sample
+    puts 'There are no configured irs_attempt_api_auth_tokens' if auth_token.nil?
+    private_key_path = 'keys/attempts_api_private_key.key'
```

Comment:
This was set at the class level earlier but Zach (rightly) pulled some of it out since it was spamming rake output in prod: https://github.com/18F/identity-idp/commit/e82e728b6ecbb2ff0fb67a4022d39ab169b4754a

We expected `auth_token` and `private_key_path` to be defined here. I'm duplicating them, and keeping the puts rather than adding a separate check. I don't think we need to worry about DRYing this up further.

Reply:
whoops great catch, thanks